src/app/organization/employees/create-employee/create-employee.component.scss

```diff
@@ -1,7 +1,7 @@
 .container {
-  max-width: 37rem;
-}
-
+    max-width: 37rem;
+  }
+  
 .loan-officer {
-  padding: 17.5px 0 17.5px 0;
-}
+    padding: 17.5px 0 17.5px 0;
+  }
```

Comment:
@DivineWisdom Remove these unnecessary changes. The linting and indentation are kept in mind and well maintained while writing the code. Please avoid making such changes in the files.

Reply:
@muskankhedia  sorry for the disorder but due to the way my code editor (VS Code) was formatted, it automatically changed the structure of the original code. I have addressed that issue though.

Reply:
Okay, please revert it.